tests/buildenv.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
is bash really required here? Try to avoid bashisms as long as possible to enhance portability.

The script doesn't do anything!

we should be able to check if we have a python interpreter and can import the stddeb stuff that we need from setup yeah?

probably easiest to do something like creating a python script and running it (or just copying a script if it is more then one line)

e.g.
```sh
#!/bin/sh

set -eux

TEST_FILE=$(mktemp --suffix=py)

cleanup() {
   if [ -e "${TEST_FILE}" ]; then
        rm "${TEST_FILE}"
}
trap cleanup EXIT

cat <<-EOT > ${TEST_FILE}
    import stdeb or whatever
    import the other thing

    maybe run something (init for example)
EOT

exit 0
```

echo "Checking python environment"
python3 "${TEST_FILE}"
(double check if the script returning an error aborts the script)